docs/monorepo.md

```diff
@@ -1,77 +1,323 @@
-**Overview**
-# The Catalyst monorepo
+**Developers**
 
-## Primary contents
+# Getting started with Catalyst
+
+How you get started with Catalyst depends on your specific goals. You might be:
+
+1. A **Storefront Developer**, looking to create a custom storefront using Catalyst as your foundation.
+2. An **Integration Developer**, aiming to build add-ons or integrations that can be optionally included in Catalyst storefronts.
+3. A **Contributor**, interested in enhancing the Catalyst project by submitting pull requests for new features, bug fixes, or other improvements.
+
+Regardless of your role, the [main GitHub repository for Catalyst](https://github.com/bigcommerce/catalyst) is the ideal starting point.
+
+The initial setup steps are the same for all developers, and this guide will later provide tailored instructions based on your specific path.
+
+## Overview
 
 The [main GitHub repository for Catalyst](https://github.com/bigcommerce/catalyst) is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) with multiple codebases, including but not limited to the following:
 
-* The [core Next.js reference storefront application, in /core](https://github.com/bigcommerce/catalyst/tree/main/core)
-* [The component library, in /core/components/ui](https://github.com/bigcommerce/catalyst/tree/main/core/components/ui)
-* [The API client, in /packages/client](https://github.com/bigcommerce/catalyst/tree/main/packages/client)
-* [Product documentation, in /docs](https://github.com/bigcommerce/catalyst/tree/main/docs)
+- [The core Next.js reference storefront application, in `/core`](https://github.com/bigcommerce/catalyst/tree/main/core)
+- [Packages published to NPM, in `/packages`](https://github.com/bigcommerce/catalyst/tree/main/packages/client)
```

Comment:
This also includes private packages too, so maybe we could reword this?

Reply:
QQ - it doesn't currently contain private packages right? You're just wanting to explain that a `packages/` folder in a monorepo can (generally speaking) contain private packages?